gestionnaire_de_stage_frontend/src/components/ManagerView/OffersNotValidView/OfferNotValidView.jsx

```diff
@@ -20,6 +20,7 @@ export default function OfferNotValidView() {
     return (
         <div className='container'>
             <h2 className="text-center">Offres de Stage non validées</h2>
```

Comment:
```suggestion
            <h2 className="text-center mb-3">Offres de Stage non validées</h2>
```

à la place du `<br/>`

Reply:
done